### ISSUE TITLE
feat request: add support for languages other than English

### ISSUE THREAD

**Issue by NotYourAverageGamer** [ChrisTitusTech/winutil#3128]
## Is your feature request related to a problem? Please describe
currently, i see no reasonable way for non-English speakers/readers to easily use `Winutil`

## Describe the solution you'd like
something like a dropdown box somewhere that would allow users to select their language, and of course allow the community to make contributions for better language support 

## Issue validation
- [x] I checked for duplicate issues.
- [x] I checked for already existing discussions.
- [x] I checked for an already existing pull request addressing the issue.

**Comment by swefd** [comment]
This is still relevant. I wanted to share this tool with my friend, but then I realized he would not understand how to use it :D

We just need a way to contribute.

**Comment by NotYourAverageGamer** [comment]
Bump

**Comment by vagman** [comment]
Still relevant. I'm highly interested in implementing the Greek translation.

**Comment by swefd** [comment]
@ChrisTitusTech Hi! Have a look at this old but still relevant issue.

**Comment by NotYourAverageGamer** [comment]
Still relevant :/

**Comment by ghost** [comment]
> Still relevant :/

yep i think @ChrisTitusTech should close dupes of this than make the first issue keep open

**Comment by ghost** [comment]
dupe of #2993 

/close
